readthedocs/projects/views/private.py

```diff
@@ -344,6 +346,31 @@ class ImportView(PrivateViewMixin, TemplateView):
     template_name = 'projects/project_import.html'
     wizard_class = ImportWizardView
 
+    def get(self, request, *args, **kwargs):
+        '''Display list of repositories to import
+
+        Adds a warning to the listing if any of the accounts connected for the
+        user are not supported accounts.
+        '''
+        deprecated_accounts = (
```

Comment:
Do these SocialAccount's get removed when they set up their new account? Or are we expecting them to "unlink" them?